ci_framework/roles/cifmw_cephadm/defaults/main.yml

```diff
@@ -64,3 +64,4 @@ cifmw_cephadm_single_host_defaults: false
 cifmw_cephadm_extra_args: ""
 cifmw_cephadm_pacific_filter: "16.*"
 cifmw_cephadm_dashboard_enabled: false
+cifmw_cephadm_basedir: "{{ cifmw_basedir | default(ansible_user_dir ~ '/ci-framework-data') }}"
```

Comment:
For now this playbook is not bound to the main ci execution, and I can run it via:

`ansible-playbook -i <inventory> ~/ci-framework/ci_framework/playbooks/ceph.yaml -e @extra.yaml` 

where I can point to a diff `cifmw_cephadm_basedir`.

Reply:
OK, good. Thanks for checking that as having it unbound is useful.